spec/services/automated_compliance/alr_validator_spec.rb

```diff
@@ -0,0 +1,31 @@
+require "rails_helper"
+
+RSpec.describe AutomatedCompliance::AlrValidator do
+  # let!(:requirement_template) {create(:requirement_template)}
+
+  # context "a PID on ALR land" do
+  #   let!(:permit_application) {create(:permit_application, requirement_template: requirement_template, pid: "006580548")}
```

Comment:
We should think about how permit applications associate with requirement templates.
Does it make sense to assign requirement templates directly to permit applications in this way?
or should the requirement template be a computed value based on the permittype, activity, and jurisdiction?

Maybe there should be a JurisdictionRequirementTemplate that belongs to a Jurisdiction and requirement template, and then instead of it being permit_application.requirement_template it is permit_applicaiton.jurisdiciton_requirement_template, which looks up the correct JurisdictionRequirementTemplate on the jurisdiciton based off of the permit_type and activity?